net.xy.ykhwong.dosbox-daum.yml

```diff
@@ -0,0 +1,92 @@
+app-id: net.xy.ykhwong.dosbox-daum
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```

Comment:
this runtime is EOL (or wlll be very soon)

23.08 is where it's at.

Reply:
it doesn't build with 23.08 and this particular fork of dosbox is a nightmare to build as it is already

Reply:
we won't accept submission on an EOL runtime.

Reply:
right, so I'm wasting my time then?

Reply:
It appears to build with 22.08, will you accept that? I still need to test it though.